config/main.py

```diff
@@ -1991,7 +1991,7 @@ def pfc(ctx):
 
 
 #
-# 'pfc asymmetric' command
+# 'pfc config asymmetric' command
```

Comment:
Change to:

```
# 'pfc asymmetric' command ('config interface pfc asymmetric ...')
```

Also change line 1983 above to:

```
# 'pfc' subgroup ('config interface pfc ...')
```

Also, after this PR is merged, can you open a new PR to move all of these PFC commands to be with the rest of the `config interface ...` commands? At some point, the commands changed from `config pfc ...` to `config interface pfc ...`.

Reply:
@jleveque Fixed